dev/reference/tools/data-migration/relay-log.md

```diff
@@ -85,7 +85,7 @@ The data purge methods for the relay log include automatic purge and manual purg
 
 ### Automatic data purge
 
-Automatic data purge includes three configuration items in the DM-worker configuration file:
+You can use the following three command-line options to change the automatic data purge strategy of DM-worker:
```

Comment:
Insert a line before this paragraph:

You can configure the automatic data purge strategy of DM-worker using the following two methods: